README.md

```diff
@@ -1 +1,10 @@
-# identity-workflow-mgt-bps-impl
\ No newline at end of file
+# identity-workflow-mgt-bps-impl
+
+
+Welcome to the WSO2 Identity Server (IS) workflow connector.
+
+The workflow connector enables you to add more control and constraints to the tasks executed within it. For instance, with the workflow feature, you can add another constraint to the ‘User Add’ operation in the WSO2 Identity Server, that requires manager approval when adding a user account. These are workflows where the operations go through a predefined path.
```

Comment:
This line doesn't make sense. Please validate.

Reply:
Resolved in [d32d623](https://github.com/wso2-extensions/identity-workflow-impl-bps/pull/77/commits/d32d6233f9a5bac746278068069c9ccc5fab25d8)